tree-construction/tests20.dat

```diff
@@ -514,3 +514,57 @@
 |       <math annotation-xml>
 |         encoding=" text/html "
 |     <div>
+
+#data
+<math><annotation-xml> </annotation-xml>
+#errors
```

Comment:
I kinda gave up on the errors once things mostly worked ok (surprise!). That's a good idea for a lil binary though, I'll tuck that under my hat and maybe one day the errors will rain down on us all.